manifests/database.pp

```diff
@@ -23,7 +23,7 @@
     logoutput   => 'on_failure',
     user        => 'apache',
     refreshonly => true,
-    require     => File['/etc/pulp/server.conf'],
```

Comment:
I suppose, is there a reason this is an include, and not between config + service?

 https://github.com/Katello/puppet-pulp/blob/master/manifests/init.pp#L377